app/models/job/state_machine.rb

```diff
@@ -1,39 +1,39 @@
-class Job
-  module StateMachine
-    def transitions
-      @transitions ||= load_transitions
-    end
+module Job::StateMachine
+  extend ActiveSupport::Concern
```

Comment:
This is an annoying diff... note the change here is just the above two lines (rest is indentation)